installer/conf/omsagent.conf

```diff
@@ -12,6 +12,25 @@
   run_interval 20m
 </source>
 
+#<source>
```

Comment:
can you create a conf in the omsagent.d for changetracking? ideally, it should be enabled by the change tracking DSC module.